urbackup.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "UrBackup",
+    "plugin_schema": 2,
```

Comment:
The `plugin_schema` has been removed from the plugin schema in https://github.com/ix-plugin-hub/iocage-plugin-index/pull/220 so guess we can remove it here now as well (and good to merge the latest `master` into this to see if the manifest looks good according to the latest `iocage` schema.